### PR TITLE
feat: hive mind panel

### PR DESCRIPTION
**Pull request by TobiTenno**
**Summary:**
Hive Mind panel!

---
**Fixes issue (include link):**
N/a

---
**Mockups, screenshots, evidence:**

image

### CODE REVIEW

File: src/views/Timer.vue [hunk 1]
```diff
@@ -93,6 +94,7 @@ export default {
       worldstate: 'worldstate',
       ostron: 'ostronSyndicate',
       solaris: 'solarisSyndicate',
+      hivemind: 'hivemindSyndicate'
```
Comment (EricSihaoLin) [inline src/views/Timer.vue:97]: Codacy Issue found: [Insert `,`](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5458165)

**Comment by EricSihaoLin** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/WFCD/warframe-hub/pullRequest?prid=5458165&bid=17890689)

**Review by EricSihaoLin — APPROVED** [review]
The only thing I would add is a small disclaimer that rewards may not be accurate. We can remove that once the bounty rewards are finalized. It is good to go though

**Comment by TobiTenno** [comment]
yeah, but that's kind of a disclaimer for everyone all the time, lol. we might want to add something to that effect in the about/license modal

**Comment by TobiTenno** [comment]
i'll put in a note to add that at some point in our issues

**Pull request merged**